bld/namelist_files/use_cases/ssp585_cam6_noresm_tropstratchem.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!--micro_do_sb_physics>   .true.   </micro_do_sb_physics-->
+<!--do_tms>                .false.  </do_tms-->
+<!--do_beljaars>           .true.   </do_beljaars-->
+
+<history_aerosol>.true.</history_aerosol>
+<modal_strat_sulfate>.true.</modal_strat_sulfate>
+<lght_no_prd_factor hgrid="0.9x1.25"> 1.0 </lght_no_prd_factor>
+<lght_no_prd_factor hgrid="1.9x2.5"> 10.0 </lght_no_prd_factor>
+
+<!-- Solar constant -->
+<solar_irrad_data_file>atm/cam/solar/SolarForcingCMIP6_18491230-22991231_c171031.nc</solar_irrad_data_file>
+
+<!-- 1850 GHG values Taken from Meinshausen et al., 2017 -->
+<flbc_type>SERIAL</flbc_type>
+<flbc_file>atm/waccm/lb/LBC_20140116-25001216_CMIP6_SSP585_0p5degLat_c180905.nc</flbc_file>
```

Comment:
Should this file path have `$INPUTDATA_ROOT`?
```suggestion
<flbc_file>$INPUTDATA_ROOT/atm/waccm/lb/LBC_20140116-25001216_CMIP6_SSP585_0p5degLat_c180905.nc</flbc_file>
```

Reply:
Hi Steve, maybe we need a common strategy on this.

Reply:
Hi, currently I suggest not to introduce $INPUTDATA_ROOT.